pyproject.toml

```diff
@@ -41,7 +40,8 @@ dependencies = [
     "rich",
     "morecantile",
     "mapbox_vector_tile",
-    "aiopmtiles @ git+https://github.com/developmentseed/aiopmtiles",
+    #"aiopmtiles @ git+https://github.com/developmentseed/aiopmtiles",
+    "aiopmtiles",
```

Comment:
I guess you need to revert this since `aiopmtiles` is not registered to pypi...that is why CI is failing...